docs/content/docs/connectors/datastream/kinesis.md

```diff
@@ -29,9 +29,26 @@ under the License.
 
 The Kinesis connector provides access to [Amazon AWS Kinesis Streams](http://aws.amazon.com/kinesis/streams/).
 
-To use the connector, add the following Maven dependency to your project:
-
-{{< artifact flink-connector-kinesis >}}
+To use this connector, add one or more of the following dependencies to your project, depending on whether you are reading from and/or writing to Kinesis Data Streams:
```

Comment:
I know the intention is to drop the Kinesis Consumer Library in the future, but I don't think that work has been prioritised yet and so very unlikely make it in time for 1.15.

Reply:
@dannycranmer Can/should we at least make sure that the Kinesis Consumer Library is marked as deprecated in Flink 1.15, so it can safely be removed in Flink 1.16+?

Reply:
@MartijnVisser is it ok to Deprecate something without providing an alternative? This might confuse developers in the meantime. I would opt to deprecate one we have a firmer ECD on the new source

Reply:
@dannycranmer You can include a reference that the successor is expected in Flink 1.16 at latest of course, but I'm more worried that if you don't deprecate it now that you will have both the old and the new connector in the future which might make things even more hard. We also want to move forward to mark the Source- and Sinkfunction as deprecated.

Reply:
ok, @CrynetLogistics can you please update the deprecation task to include the source?